### DISCUSSION TITLE
Why is the Docker Hub container not a part of the recommended deployment setup?

### DISCUSSION THREAD

**Discussion by tecoholic** [misskey-dev/misskey#9222] [Q&A] [answered]
Hi, 

Thank you for the team that has created this software. I recently have created my own instance following the [documentation here](https://misskey-hub.net/en/docs/install/docker.html). While moving the instance from VM to my personal server, I had to rebuild the image again. This also meant, I couldn't use [Nomad](https://www.nomadproject.io/) to deploy it, at least not in a straightforward manner.

After searching, I found that there are actually versioned images in Docker Hub for [misskey/misskey](https://hub.docker.com/r/misskey/misskey), including the `develop` one published just 5 days ago. I am wondering why using the image is not a part of the recommended deployment instructions?

If using the Docker Hub image is a valid method of deployment, and it is a part of the [Roadmap](https://github.com/misskey-dev/misskey/blob/develop/ROADMAP.md) to update the documentation, I will be willing to volunteer for updating the docs. But, I would like to figure out the reasons before opening an issue.

**Comment by ThatOneCalculator** [comment, marked as answer]
The docker image should be fine, most likely just incomplete documentation

**Reply by tecoholic** [reply]
You are right. I redeployed using the docker image and everything is working as expected :)

**Reply by chualuoi** [reply]
@tecoholic: Would you mind updating the documentation?

**Reply by tecoholic** [reply]
@chualuoi I will be happy to. Let me create a PR, when I get some free time this week :)

**Comment by tassoman** [comment]
@syuilo San, is it possible to tag *13.0.0-beta* commits also?
For [example 13.0.0-beta.14](https://github.com/misskey-dev/misskey/commit/a4a614e18092446e658bb6b6b8cb0921c52e286d) :thinking:
